src/data/index.js

```diff
@@ -2139,6 +2139,15 @@ const questions = [
     premium: false,
     companies: ['Amazon', 'Microsoft', 'Bloomberg', 'Apple', 'Adobe'],
   },
+  {
+    id: 2022,
```

Comment:
Please change this to `170` - the `id` field isn't the id of the leetcode question in this case

Reply:
Hi @seanprashad , thanks for the info. I have updated the pull req. Kindly please merge it.